deblur/workflow.py

```diff
@@ -52,9 +54,33 @@ def multiple_sequence_alignment(seqs_fp):
     pass
 
 
-def remove_chimeras_denovo_from_seqs(seqs_fp):
-    """Step 5: remove chimeras de novo using UCHIME"""
-    pass
+def remove_chimeras_denovo_from_seqs(seqs_fp, output_fp):
+    """Step 5: remove chimeras de novo using UCHIME
```

Comment:
Remove "Step 5" and "(VSEARCH implementation)". The first sentence in the numpy doc should be a single line. You can add then another block providing more in depth description.